api/api-referential/referential-commons/src/main/java/fr/gouv/vitamui/referential/common/rest/RestApi.java

```diff
@@ -73,6 +73,8 @@
 
     public static final String ACCESSION_REGISTER_URL = PREFIX + "/accession-register";
 
+    public static final String ACCESSION_REGISTER_DETAIL_URL = PREFIX + "/accessionregisterdetails";
+
```

Comment:
Pourquoi avoir créer un autre endpoint si c'est juste pour récupérer des informations supplémentaires "details". On aurait pu utiliser le même et ajouter un sufixe. 

La convention serait plutôt d'utiliser spinal-case : 
It is recommended to use the spinal-case (which is highlighted by RFC3986), this case is used by Google, PayPal, and other big companies.

Reply:
Regroupement fonctionnel fait autour de l'URL 'accession-register' pour accéder aux données des collections (ACCESSION_REGISTER_SUMMARY et ACCESSION_REGISTER_DETAIL)